certificate-management.yaml

```
@@ -0,0 +1,346 @@
+      responses:
+        "200":
+          description: "OK"
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/EncryptedPrivateKey'
+        "400":
+          description: | 
+            Bad Request  
+            types: MalformedLoginToken, MultipleAuthorizationError
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/GenericError'
+        "401":
+          description: |
+            Unauthorized  
+            types: JwtAuthorization, LoginTokenExpired
+          content:
+            application/json:  
+              schema:
+                $ref: '#/components/schemas/GenericError'
+        "403":
+          description: |
+            Forbidden  
+            types: OwnerMismatch
+          content:
+            application/json:  
+              schema:
+                $ref: '#/components/schemas/GenericError'
+        "404":
+          description: |
+            NotFound  
+            types: NotEnrolled
+          content:
+            application/json:  
+              schema:
+                $ref: '#/components/schemas/GenericError'
+components:
+  securitySchemes:
+    uc4_token_login:
+      type: apiKey
+      in: cookie
+      name: login
+    uc4_token_login_header:
+      type: http
+      scheme: bearer
+  schemas:
+    EncryptedPrivateKey:
+      type: "object"
+      properties:
+        key:
+          type: string
+          description: "The encrypted private key."
+        iv:
+          type: string
+          description: "The IV for key encryption/decryption."
+        salt:
+          type: string
+          description: "The salt used to derive the key from the password."
+        
+    PostMessageCSR:
+      type: "object"
+      properties:
+        certificateSigningRequest:
+          type: string
+          description: "In PEM format, x509 certificate signing request, as specified in PKCS#10. Common-name must be the enrollmentId."
+        encryptedPrivateKey:
+          allOf:
+            - $ref: "#/components/schemas/EncryptedPrivateKey"
+    JsonCertificate:
+      type: "object"
+      properties:
+        certificate:
+          type: string
+          description: "In PEM format, x509 certificate, as specified in PKCS#10."
+    JsonEnrollmentId:
+      type: "object"
+      properties:
+        id:
+          type: string
+    JsonVersionNumber:
+      type: object
+      properties:
+        versionNumber:
+          type: string
+    GenericError:
+      allOf:
+        - $ref: 'https://raw.githubusercontent.com/upb-uc4/api/develop/errors.yaml#/components/schemas/GenericError'
+        - type: object
+    DetailedError:
+      allOf:
+        - $ref: 'https://raw.githubusercontent.com/upb-uc4/api/develop/errors.yaml#/components/schemas/DetailedError'
+        - type: object
+        
+  examples:
+    PostMessageCsrExample:
+      summary : "descr"
```

```suggestion
      summary : "PostMessageCsr Example"
```
This looks very weird in the editor. You may use my suggestion or write a small summary yourself. Note that this is the text that appears in the example dropdown, so it should be short.